src/styles/blocks/button.scss

```diff
@@ -0,0 +1,31 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.button {
```

Comment:
https://prnt.sc/NkE4D6PAZlFK => button for form has border, I think it is better to do all button in the same style